include/evmc/evmc.h

```diff
@@ -11,6 +11,7 @@
 #ifndef EVMC_H
 #define EVMC_H
 
+#include <stdbool.h>
```

Comment:
Not sure, but could add `Definition of bool, true, false` here :)

Reply:
It seemed self explained.

Reply:
I added the comment.